lib/spack/spack/build_systems/cmake.py

```diff
@@ -38,19 +38,36 @@ class CMakePackage(PackageBase):
 
     This class provides three phases that can be overridden:
 
-    * cmake
-    * build
-    * install
+        1. :py:meth:`~.CMakePackage.cmake`
+        2. :py:meth:`~.CMakePackage.build`
+        3. :py:meth:`~.CMakePackage.install`
 
     They all have sensible defaults and for many packages the only thing
-    necessary will be to override ``cmake_args``
+    necessary will be to override :py:meth:`~.CMakePackage.cmake_args`.
+    For a finer tuning you may override:
```

Comment:
Why is `cmake_args()` not described in the table below?  Is the most important helper method getting lost?